From: Dhaval Vasa <dhaval.vasa@einfochips.com>
Date: Fri, 7 Aug 2009 11:56:49 +0000 (+0530)
Subject: USB: ftdi_sio: add product_id for Marvell OpenRD Base, Client
X-Git-Tag: v2.6.31-rc6~35^2~2
X-Git-Url: http://git.kernel.org/?p=linux%2Fkernel%2Fgit%2Ftorvalds%2Flinux-2.6.git;a=commitdiff_plain;h=50d0678e2026c18e4147f0b16b5853113659b82d

USB: ftdi_sio: add product_id for Marvell OpenRD Base, Client

reference:
http://www.open-rd.org

Signed-off-by: Dhaval Vasa <dhaval.vasa@einfochips.com>
Cc: stable <stable@kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


diff -urN a/drivers/usb/serial/ftdi_sio.c b/drivers/usb/serial/ftdi_sio.c
--- a/drivers/usb/serial/ftdi_sio.c	2009-09-25 21:55:41.000000000 +0000
+++ b/drivers/usb/serial/ftdi_sio.c	2009-09-25 21:57:21.000000000 +0000
@@ -645,6 +645,8 @@
 	{ USB_DEVICE(FTDI_VID, FTDI_REU_TINY_PID) },
 	{ USB_DEVICE(MARVELL_VID, MARVELL_SHEEVAPLUG_PID),
 		.driver_info = (kernel_ulong_t)&ftdi_jtag_quirk },
+	{ USB_DEVICE(MARVELL_VID, MARVELL_OPENRD_PID),
+		.driver_info = (kernel_ulong_t)&ftdi_jtag_quirk },
 	{ },					/* Optional parameter entry */
 	{ }					/* Terminating entry */
 };
diff -urN a/drivers/usb/serial/ftdi_sio.h b/drivers/usb/serial/ftdi_sio.h
--- a/drivers/usb/serial/ftdi_sio.h	2009-09-25 21:55:20.000000000 +0000
+++ b/drivers/usb/serial/ftdi_sio.h	2009-09-25 21:58:05.000000000 +0000
@@ -870,6 +870,13 @@
 #define MARVELL_SHEEVAPLUG_PID 0x9e8f
 
 /*
+ * Marvell OpenRD Base, Client
+ * http://www.open-rd.org
+ * OpenRD Base, Client use VID 0x0403
+ */
+#define MARVELL_OPENRD_PID	0x9e90
+
+/*
  *   BmRequestType:  1100 0000b
  *   bRequest:       FTDI_E2_READ
  *   wValue:         0

